### PR TITLE
Update stringtable.xml

### PR DESCRIPTION
**Pull request by GoldJohnKing**
**Added Simplified Chinese Translation**

- Add: Simplified Chinese Translation in stringtable.xml (@GoldJohnKing).

**When merged this pull request will:**
- This pull request adds Simplified Chinese translation in stringtable.xml. Though I've tried my best to make the translation both precise and easy to understand, there're still some of them that I'm not sure. So I just chose the most accurate translation to me, and added a tag there, hoping other Chinese speaker developers helps to make it for sure.
- ref https://github.com/Vdauphin/HeartsAndMinds/issues/400

**Final test:**
- Already double-checked the translations and xml tags, tested on both dedicated server and client server, played through out personally, everything should be fine.
- [x] local
- [x] server

**Screenshots**
- None.

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 1]
```diff
@@ -229,12 +258,15 @@
                 <Spanish>Equipamiento (Arsenal)</Spanish>
                 <German>Ausrüstung (Arsenal)</German>
                 <Portuguese>Equipamento (Arsenal)</Portuguese>
+                <Chinesesimp>装备(军火库)</Chinesesimp>
+
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: Remove the line

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 2]
```diff
@@ -1433,69 +1661,82 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_O_FOB_CREATE_H_WIP">
                 <Original>Get back! Mounting FOB</Original>
                 <Spanish>¡Atrás! Montando FOB</Spanish>
                 <German>Geh ein wenig zurück! Die FOB wird aufgebaut</German>
                 <Portuguese>Afaste-se! Montando a FOB</Portuguese>
+                <Chinesesimp>回去! FOB装配中</Chinesesimp>
             </Key>
-            <Key ID="STR_BTC_HAM_O_FOB_DISMANTLE_H_PROC">
+            <Key ID="STR_BTC_HAM_O_FOBDISMANTLE__H_PROC">
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: Wrong ID, you added a `_` in the ID name

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 3]
```diff
@@ -1433,69 +1660,82 @@
...
             </Key>
             <Key ID="STR_BTC_HAM_O_FOB_CREATE_H_WIP">
                 <Original>Get back! Mounting FOB</Original>
                 <Spanish>¡Atrás! Montando FOB</Spanish>
                 <German>Geh ein wenig zurück! Die FOB wird aufgebaut</German>
                 <Portuguese>Afaste-se! Montando a FOB</Portuguese>
+                <Chinesesimp>FOB装配中, 请远离!</Chinesesimp>
             </Key>
-            <Key ID="STR_BTC_HAM_O_FOB_DISMANTLE_H_PROC">
+            <Key ID="STR_BTC_HAM_O_FOBDISMANTLE_H_PROC">
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: Still wrong ID name x)
```suggestion
            <Key ID="STR_BTC_HAM_O_FOB_DISMANTLE_H_PROC">
```

Comment (GoldJohnKing) [reply =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: This time it should be ok...

**Pull request merged**